add.go

```diff
@@ -456,9 +456,11 @@ func (b *Builder) Add(destination string, extract bool, options AddAndCopyOption
 		// Iterate through every item that matched the glob.
 		itemsCopied := 0
 		for _, glob := range localSourceStat.Globbed {
-			rel, err := filepath.Rel(contextDir, glob)
-			if err != nil {
-				return fmt.Errorf("computing path of %q relative to %q: %w", glob, contextDir, err)
+			rel := glob
+			if filepath.IsAbs(glob) {
```

Comment:
Is this correct?  We only append contextDir, if the glob is a Absolute path?

Reply:
It isn't appended - we're conditionalizing whether or not we want to convert the matched value to a relative path.  We don't need to do that when it's already a relative path.

Reply:
Ok, now I understand.